UnitTests/GitUI.Tests/UserControls/CommitInfo/BranchComparerTests.cs

```diff
@@ -7,26 +8,35 @@ public class BranchComparerTests
     {
         [SetCulture("en-US")]
         [SetUICulture("en-US")]
-        [Test]
-        public void BranchComparer([Values(null, "", "current", "(no branch)")] string? currentBranch)
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase("current")]
+        [TestCase("(no branch)")]
+        public void BranchComparer(string? currentBranch)
```

Comment:
(`Values` is useful, particularly if having multiple arguments (creates the cross-product).)

Reply:
I was tinkering with some more test cases, this was clearer. I can reverse.

Reply:
> I was tinkering with some more test cases, this was clearer. I can reverse.

No need to reverse. That's why I set it in parens. I just wanted to mention it.